content/sensu-go/6.5/sensu-plus.md

```diff
@@ -0,0 +1,198 @@
+For more information, read [Get started with commercial features](../commercial/).
+{{% /notice %}}
+
+Sensu Plus is a built-in integration you can use to transmit your Sensu observability data to Sumo Logic via the Sumo Logic [HTTP Logs and Metrics Source][1].
+You can use Sumo Logic's interactive dashboards and analytics tools to get better visibility into your Sensu data.
+
+To use Sensu Plus, you need a [Sumo Logic account][2].
+Once you have an account, follow this guide to start sending your Sensu data to Sumo Logic.
+
+## Set up an HTTP Logs and Metrics Source
+
+Set up a Sumo Logic HTTP Logs and Metrics Source to collect your Sensu observability data:
+
+1. In the Sumo Logic left-navigation menu, click **Manage Data** and **Collections**.
+
+    {{< figure src="/images/sumo-plus/manage-data_collection.gif" alt="Open the Collections tab" link="/images/sumo-plus/manage-data_collection.gif" target="_blank" >}}
+
+2. At the top-right of the Collection tab, click **Add Collector**.
+
+    {{< figure src="/images/sumo-plus/add-collector.png" alt="Add a Sumo Logic collector" link="/images/sumo-plus/add-collector.png" target="_blank" >}}
+
+3. In the Click Selector Type modal window, click **Hosted Collector**.
+
+    {{< figure src="/images/sumo-plus/hosted-collector.png" alt="Select the hosted collector option" link="/images/sumo-plus/hosted-collector.png" target="_blank" >}}
+
+4. In the Add Hosted Collector modal window:
+    - Type **sensu** in the Name field.
+    - Click **Save**.
+
+    {{< figure src="/images/sumo-plus/add-hosted-collector.png" alt="Name the hosted collector" link="/images/sumo-plus/add-hosted-collector.png" target="_blank" >}}
+
+5. In the Confirm prompt, click **OK**.
+
+    {{< figure src="/images/sumo-plus/confirm-prompt.png" alt="Confirm your hosted collector configuration" link="/images/sumo-plus/confirm-prompt.png" target="_blank" >}}
+
+6. Under Cloud APIs, click **HTTP Logs & Metrics**.
+
+    {{< figure src="/images/sumo-plus/cloud-apis_http-logs-and-metrics.png" alt="Select the HTTP Logs & Metrics source" link="/images/sumo-plus/cloud-apis_http-logs-and-metrics.png" target="_blank" >}}
+
+7. In the HTTP Logs & Metrics form:
+    - Type **sensu-http** in the Name field.
+    - Type **sensu-events** in the Source Category field.
+    - Click **Save**.
+
+    {{< figure src="/images/sumo-plus/http-logs-and-metrics_source.gif" alt="Select options for HTTP Logs & Metrics source" link="/images/sumo-plus/http-logs-and-metrics_source.gif" target="_blank" >}}
+
+8. In the HTTP Source Address prompt, copy the listed URL and click OK.
+You will use this URL as the value for the `url` attribute in your [Sumo Logic metrics handler][3] definition.
+
+    {{< figure src="/images/sumo-plus/http-source-address_url.png" alt="Retrieve the HTTP Source Address URL" link="/images/sumo-plus/http-source-address_url.png" target="_blank" >}}
+
+## Create a Sumo Logic metrics handler in Sensu
+
```

Comment:
I might not have this quite correct -- you can use the Sumo Logic HTTP Logs and Metrics Source URL in a regular pipe handler too (like our regular Sensu Sumo Logic Handler asset), right?

Reply:
Yes